.github/workflows/release.yaml

```
@@ -0,0 +1,130 @@
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run validation and tests
+        run: npm test
+
+      - name: Bump version
+        run: npm version ${{ inputs.bump }} -m "chore(release): v%s [skip ci]"
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)